run.sh

```diff
@@ -44,7 +44,7 @@ else
     echo "=> /data/debian directory does not exist:"
     echo "   Configuring a default debian repository with reprepro..."
 
-    keyid=$(gpg --dry-run /config/reprepro_pub.gpg | sed "s/.*\/\([^ ]*\).*/\1/")
+    keyid=$(gpg --dry-run /config/reprepro_pub.gpg | grep pub | sed "s/.*\/\([^ ]*\).*/\1/")
```

Comment:
depending on the version of gpg, it outputs more additional text, didn't work for me without this grep